style.css

```
@@ -0,0 +1,188 @@
+body{
+    background-color: black
+}
+
+.container{
+    position: absolute;
+    width: 26rem;
+    left: 50%;
+    top : 10%;
+    margin-left: -13rem;
+    height: 80%;
+    border-style: solid;
+    border-radius: 2%;
+    border-color: white;
+    align-items: center;
+}
+
+.title{
+    width: 90%;
+    margin-left: 5%;
+    height: 38px;
+    color: white;
+    font-size: 27px;
+    text-align: center;
+    font-weight: 800;
+
+    border-bottom: solid;
+    border-radius: 2%;
+}
+
+.dataInput{
+    margin-top: 1.5rem;
+    position: absolute;
+    margin-left: 10%;
+    width:80%;
+    height: 30px;
+    border-style: solid;
+    border-width: 1px;
+    border-color: white;
```

요기 `border`과 관련된 속성 (border-radius제외)는 아래와 같이 작성하면 한줄로 작성할 수 있습니다!
```suggestion
    border: 1px solid white;
```